imports/ui/unverified-warning/unverified-warning.actions.js

```diff
@@ -0,0 +1,10 @@
+export const RESEND_VERIFICATION = 'resend_verification'
+export const ERROR = 'resend_verification_error'
+export const SUCCESS = 'resend_verification_success'
+
+export function resendVerification (userId) {
+  return {
+    type: RESEND_VERIFICATION,
+    userId
```

Comment:
You don't need to pass the userId as a part of the action. The type alone should suffice